packages/mui-docs/src/NProgressBar/NProgressBar.js

```diff
@@ -50,7 +50,7 @@ function NProgressBar(props) {
             height: 2,
             zIndex: (theme.vars || theme).zIndex.tooltip,
             backgroundColor: (theme.vars || theme).palette.primary[200],
-            ...theme.applyDarkStyles({
```

Comment:
@DiegoAndai The root cause of the template error comes from this component.

The NProgressBar is used in all of the joy templates. It try to access theme.applyDarkStyles but the branding theme was removed from the global _app.js so the error appear.